tests/common.py

```diff
@@ -21,12 +22,18 @@
 }
 
 TESTDATA["solution.rvc"] = TD / "solution.rvc"
-TESTDATA["raven-gr4j-cemaneige-nc-ts"] = (
-    TD / "raven-gr4j-cemaneige" / "Salmon-River-Near-Prince-George_meteo_daily.nc"
-)
-TESTDATA["raven-gr4j-cemaneige-nc-rv"] = tuple(
-    (TD / "raven-gr4j-cemaneige").glob("raven-gr4j-salmon.rv?")
-)
+TESTDATA[
+    "raven-gr4j-cemaneige-nc-ts"
+] = "raven-gr4j-cemaneige/Salmon-River-Near-Prince-George_meteo_daily.nc"
+TESTDATA["raven-gr4j-cemaneige-nc-rv"] = query_folder("raven-gr4j-cemaneige", pattern=".rv")
```

Comment:
@cjauvin do you think for the test suite we should replace the TESTDATA dict by pytest fixtures calling get_file and query_folder ?